langchaingo/README.md

```diff
@@ -0,0 +1,117 @@
+# 🧠 Langchaingo + DuckDuckGo with Model Context Protocol (MCP)
+
+This project demonstrates a **zero-config application** using [Langchaingo] and
+the **Model Context Protocol (MCP)** to answer natural language questions by performing
+real-time web search via [DuckDuckGo] — all orchestrated with [Docker Compose].
+
+> [!Tip]
+> ✨ No configuration needed — run it with a single command.
+
+<p align="center">
+  <img src="demo.mov"
```

Comment:
question: does this render for you in the GH README preview? It doesn't seem to work for me:

<img width="1020" height="452" alt="Image" src="https://github.com/user-attachments/assets/b850b692-b75e-43a6-93ad-b65ee131d661" />

For other demos we've used ffmpeg to convert the video to a gif (with something `ffmpeg -i file.mov  -vf "setpts=0.5*PTS,fps=20,scale=1200:-1" -c:v gif demo.gif`)